ext/natives/codegen_cfx_natives.lua

```diff
@@ -1731,6 +1731,14 @@ native 'SET_VEHICLE_WHEEL_X_OFFSET'
 	</summary>
 ]]
 
+native 'OVERRIDE_PLAYER_TALKING'
```

Comment:
Please add a doc block with a description of the native and the arguments, similarly that it doesn't need to be called every frame.

Of course, if changing the native's name, also change the definition here. 🙂